projects/pickle/index.js

```diff
@@ -1,114 +1,92 @@
-const sdk = require("@defillama/sdk");
-const abi = require("./abi.json");
-const utils = require("../helper/utils");
-const { unwrapUniswapLPs, unwrapCrv } = require("../helper/unwrapLPs");
-const { getChainTransform } = require("../helper/portedTokens");
 const { toUSDT, usdtAddress } = require("../helper/balances");
-const { staking } = require("../helper/staking");
+const axios = require("axios");
+const retryModule = require("async-retry");
 
-const excluded = ["pbamm", "pickle-eth", "sushi-pickle-eth"];
-const jars_url =
-  "https://stkpowy01i.execute-api.us-west-1.amazonaws.com/prod/protocol/pools";
+async function retry(func) {
+  return retryModule(func, {
+    retries: 3,
+  });
+}
 
-// Contracts
-const dillAddress = "0xbBCf169eE191A1Ba7371F30A1C344bFC498b29Cf";
+const endpoints = [
+  "https://api.pickle.finance/prod/protocol/pfcore/",
+  "https://f8wgg18t1h.execute-api.us-west-1.amazonaws.com/prod/protocol/pfcore/",
```

Comment:
Nice job @MisterMard I think we can drop this fallback API endpoint to simplify things, it shouldn't really be exposed publicly. Similarly, retry logic shouldn't be necessary. DefiLlama will try again next cycle if anything fails.

Reply:
Thanks for your comments. All resolved now.